code/game/objects/structures/props.dm

```diff
@@ -1042,26 +1042,26 @@
 		helmet.forceMove(loc)
 		helmet = null
 	if(tagged)
-		var/obj/item/dogtag/new_info_tag = new(loc)
+		var/obj/item/information_tag/new_info_tag = new(loc)
 		new_info_tag.fallen_names = list(dogtag_name)
 		new_info_tag.fallen_assgns = list(dogtag_assign)
 		new_info_tag.fallen_blood_types = list(dogtag_blood)
 		fallen_list_cross -= dogtag_name
 	return ..()
 
 /obj/structure/prop/wooden_cross/attackby(obj/item/W, mob/living/user)
-	if(istype(W, /obj/item/dogtag))
-		var/obj/item/dogtag/dog = W
+	if(istype(W, /obj/item/information_tag))
+		var/obj/item/information_tag/tag = W
 		if(!tagged)
 			tagged = TRUE
 			user.visible_message(SPAN_NOTICE("[user] drapes the [W] around the [src]."))
-			dogtag_name = popleft(dog.fallen_names)
-			dogtag_assign = popleft(dog.fallen_assgns)
-			dogtag_blood = popleft(dog.fallen_blood_types)
+			dogtag_name = popleft(tag.fallen_names)
+			dogtag_assign = popleft(tag.fallen_assgns)
+			dogtag_blood = popleft(tag.fallen_blood_types)
```

Comment:
When this causes the id tags to return back to one tag, it doesn't revert the pluralization:
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/d0db7093-64a6-4911-bf3f-2b485c3f95d4)